nixos/modules/services/hardware/nvidia-container-toolkit/default.nix

```diff
@@ -0,0 +1,44 @@
+{ config, lib, pkgs, ... }:
+
+{
+
+  options = {
+
+    hardware.nvidia-container-toolkit.enable = lib.mkOption {
+      default = false;
+      type = lib.types.bool;
+      description = lib.mdDoc ''
+        Enable the NVIDIA Container Toolkit (nvidia-ctk) service that
+        generates a Container Device Interface (CDI) configuration file
+        for auto-detected NVIDIA devices, so that CDI-compliant container
+        runtimes can use the forwarded host GPU's.
```

Comment:
I'd reword this to something like: "Enable dynamic CDI configuration for NVidia devices by running `nvidia-ctk` on boot"